vsts/pipelines/templates/_securityChecks.yml

```diff
@@ -2,26 +2,40 @@ steps:
 - checkout: self
   clean: true
 
-- task: DotNetCoreCLI@2
-  displayName: 'dotnet publish'
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@1
+  displayName: 'Run PoliCheck'
   inputs:
-    command: publish
-    publishWebProjects: false
-    projects: '$(Build.SourcesDirectory)\src\**\*.csproj'
-    arguments: '-o $(Build.BinariesDirectory)'
-    zipAfterPublish: false
-    modifyOutputPath: false
+    targetType: F
+  condition: always()
 
 - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
   displayName: 'Run CredScan'
   inputs:
     debugMode: false
   condition: always()
 
+- task: NuGetCommand@2
+  displayName: 'Run "nuget restore" on Oryx solution'
+  inputs:
+    command: 'restore'
+    restoreSolution: '$(Build.SourcesDirectory)\Oryx.sln'
+
+- task: VSBuild@1
+  displayName: 'Build the Oryx solution'
+  inputs:
+    solution: '$(Build.SourcesDirectory)\Oryx.sln'
+    configuration: 'debug'
```

Comment:
Could you please clarify why we need the debug configuration? Are there any side effects of doing that?

Reply:
When we were running `dotnet publish` previously, we didn't provide a configuration, which then defaults to `debug` for the build. The documentation for the `VSBuild` task wasn't explicit about what the default configuration was if it wasn't configured, so I figured it would be better to provide it then not (though I don't think there would be an issue with doing `debug` vs. `release` in this case, the binaries should be the same).

Reply:
OK, thanks for clarifying! Sounds good, I don't see why we couldn't keep as is.